WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -91,6 +91,9 @@ extension WPStyleGuide
         public static let badgeQuotedStyle          = blockQuotedStyle
         
         //  Blocks
+        public static let contentBlockRegularFont   = WPFontManager.merriweatherLightFontOfSize(blockFontSize)
+        public static let contentBlockBoldFont      = WPFontManager.merriweatherRegularFontOfSize(blockFontSize)
```

Comment:
The goal here wasn't to expose the name of the font being used, but to indicate (as clearly as possible) what's the spot in which it should be used.

Plus we also deal with Styles dictionaries (which contain ParagraphStyle, Font and Foreground colors). Updating that with the font name wouldn't look good.